index.d.ts

```diff
@@ -84,9 +116,7 @@ declare module 'react-pixi-fiber' {
   export class Sprite extends React.Component<SpriteProperties> {}
 
   /** `Text` component properties */
-  export interface TextProperties extends ChildlessComponent<Omit<PIXI.Text, 'anchor'>> {
-    anchor?: string | number[] | PIXI.ObservablePoint;
-  }
+  export interface TextProperties extends ChildlessComponent<PIXI.Text & InteractiveComponent> {}
```

Comment:
This change removes what was done in #107, any particular reason for that?

Reply:
Oops, no idea how that happened... I've put it back in in 9f4555b